src/imu/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.5)
+project(imu)
```

Comment:
Are you sure this is useful? For sure this directory can't be built on its own (missing find_package), so probably we can get rid of this and avoid using `${PROJECT_NAME}` in the rest of the CMake code?

Reply:
I added the project name to be coherent with the other tests in `icub-test`, but if it is not necessary, I'll remove it